### ISSUE TITLE
Dropdown: allow user to disable automatic active state

### ISSUE THREAD

**Issue by sohkai** [aragon/aragon-ui#443]
In some cases, the dropdown may not want to have a default active state, with the border. Usually this would be when it's not used as a filter, but as an item selector:

<img width="715" alt="Screen Shot 2019-08-10 at 1 35 18 PM" src="https://user-images.githubusercontent.com/4166642/62821356-c7d88600-bb73-11e9-90c7-9edd2e42a898.png">

If this behaviour is only desired for filters, it may make sense to only enable it with a prop?

**Comment by bpierre** [comment]
Yes I was thinking about it recently, I think we need to have a way to express that:

- We want an item to be selected (usually the first one), not a placeholder.
- We don’t want the selected state to be visible until the user does it.

Actually that could even be the default, and we could have a prop to get the current behavior.
